caos.ansible_roles/roles/nrql_api_request/tasks/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+
+- name: Execute NRQL
+  delegate_to: 127.0.0.1
+  uri:
+    headers:
+      Content-Type: "application/json"
+      API-Key: "{{ nr_api_key }}"
```

Comment:
Should we add those variables into a vars file? I would also add them into the README, so we can easily use this role.

Reply:
nice catch! 👀 thanks!

Added to 9a15f832b0c38dbf484c55c6575e2b3793dbe834

Reply:
Great! Thanks